### PR TITLE
Update docs to better structure

### PR DESCRIPTION
**Pull request by warango4**
Pull request

### What this PR does / why we need it
Align documentation with latest changes and update for better readability.

### Which issue(s) this PR fixes

N/A

### Additional information or special notes for your reviewer

Signed-off-by: Wendy Arango <warango@vmware.com>

### CODE REVIEW

File: docs/debug-workload.md [hunk 1]
```diff
@@ -35,70 +18,121 @@ Once the workload is created, you can tail the workload to view the build and ru
...
-       No messages found.
+- *MissingValueAtPath*
+	- *Message*: Waiting to read value `[.status.artifact.url]` from resource gitrepository.source.toolkit.fluxcd.io  in namespace `[ns]`
+	- *Possible Causes*:
+		- The git `url/tag/branch/commit` params passed in the workload are not valid.
+			- *Resolution*: Fix the invalid git param by using *tanzu apps workload apply*
+		- The git repo is not accessible from the cluster
+			- *Resolution*: Configure your cluster networking or your Git repo networking so that they can communicate with each other.
+		- The namespace is missing the git secret for communicating with the private repository
+			- *Resolution*: Checkout this page on how to setup Git Authentication for Private repositories [Link to [Git authentication](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.1/tap/GUID-scc-git-auth.html)]
```
Comment (heyjcollins) [inline docs/debug-workload.md:67]: this has a 1.1 reference.

**Review by heyjcollins — DISMISSED** [review]
I approve assuming the one suggested change comment I made will be resolved.

File: docs/debug-workload.md (hunk 1 above)
Comment (warango4) [reply docs/debug-workload.md]: okay, changed to reference TAP1.3

**Review by heyjcollins — APPROVED** [review]
LGTM - happy holidays @warango4!

**Review by anibmurthy — APPROVED** [review]

**Pull request merged**